Source/WebCore/animation/KeyframeEffectStack.cpp

```diff
@@ -277,8 +277,16 @@ void KeyframeEffectStack::cascadeDidOverrideProperties(const HashSet<AnimatableP
 
 void KeyframeEffectStack::applyPendingAcceleratedActions() const
 {
-    for (auto& effect : m_effects)
-        effect->applyPendingAcceleratedActionsOrUpdateTimingProperties();
+    auto hasActiveAcceleratedEffect = m_effects.containsIf([](const auto& effect) {
```

Comment:
I don't find 'auto' useful for bool.

Shame we have to do two passes over m_effects in this function.

Reply:
> I don't find 'auto' useful for bool.

I'll use `bool` when landing.

> Shame we have to do two passes over m_effects in this function.

True. That said, it's unlikely that `m_effects` is ever containing a large number of items. These are only the [_relevant_](https://drafts.csswg.org/web-animations-1/#relevant-animations-section)  effects for a given element.